.github/workflows/release.yml

```diff
@@ -248,19 +302,20 @@ jobs:
       - name: Install dependencies and pyogrio wheel
         shell: bash
         run: |
-          python -m pip install -r ci/requirements-wheel-test.txt
+          uv pip install -r ci/requirements-wheel-test.txt
           if [ ${{ matrix.python-version }} != "3.12" ]; then
-            python -m pip install --no-deps geopandas
+            uv pip install geopandas
```

Comment:
It might not be strictly needed, but I would keep the `--no-deps` here for geopandas, just to avoid we install a released pyogrio that is then overwritten below

Reply:
Actually, it might definitely be needed, because from the logs it seems it then skips installing the local wheel because pyogrio is already installed? 

We could also swap the order to prevent this (first install the local wheel before installing geopandas, that's also the order we use for the sdist tests). But would also still use `--no-deps` to be sure.

Reply:
Whoops, yes, that was a mistake to remove `--no-deps`.  Switched order.